src/features/txnDelegation/useSolTxns.ts

```diff
@@ -100,9 +100,6 @@ const useSolTxns = (heliumAddress: string, solanaTransactions?: string) => {
 
       const sigs = await connection.getSignaturesForAddress(
         new PublicKey(keyToAssetAccount.pubkey),
-        {
```

Comment:
@ChewingGlass I need to get the latest signature here. Is it fine to just fetch all and grab the last one?

Reply:
I think this is a super complicated way of getting the information. Why not just use https://github.com/helium/helium-program-library/blob/master/packages/hotspot-utils/src/hotspotUtils.ts#L136?